bin/dbsync

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env node
+
+var path   = require('path');
+var fs     = require('fs');
+var coffee = require('coffee-script');
+var src    = path.join(path.dirname(fs.realpathSync(__filename)), '../src');
+
```

Comment:
If we want to start using coffeelint or something similar, I believe it is intended to be used as part of a build, not loaded as part of the app.  So, we can start doing that in general, but it wouldn't take place as part of this file.

Also, in order to do that, you and I would have to have a cage match to agree on the rule setup, since we seem to have very different opinions about the "right" way to do things in coffeescript.  ;-)